lua/lspconfig/server_configurations/sourcery.lua

```diff
@@ -9,6 +9,23 @@ local root_files = {
   'pyrightconfig.json',
 }
 
+local get_token_from_auth_file = function()
+  local config_home = vim.fn.getenv 'XDG_CONFIG_HOME' or vim.fn.expand '~/.config'
+  local auth_file_path = config_home .. '/sourcery/auth.yaml'
```

Comment:
why hardcode the path sep in there? it only work for *nix ?

Reply:
true I'll check what it is for windows

Reply:
https://github.com/neovim/nvim-lspconfig/pull/2240/commits/3d648fdbe1e731564438381ec1eb6ecabaf660ba